src/components/WorkOrder/WorkOrderDetails.js

```diff
@@ -41,7 +41,7 @@ const WorkOrderDetails = ({
       <GridRow>
         <GridColumn width="two-thirds">
           <h1 className="lbh-heading-h1 display-inline govuk-!-margin-right-6">
-            Work order: {workOrder.reference}
+            Work order: {workOrder.reference.toString().padStart(8, '0')}
```

Comment:
would you mind adding a test for that to WorkOrderDetails.test.js for this behaviour? (to have a snapshot of it)

Reply:
Evangelos has a good point, were you able to discuss that possibility with b/e developers?

Reply:
This is a good idea, and worth doing - though throughout the backend, the work order reference is an int - so it would need a substantial refactor. Therefore worth keeping this change too in my opinion